lib/colonel_kurtz/renderer.ex

```diff
@@ -42,49 +53,49 @@ defmodule ColonelKurtz.Renderer do
     ]
   end
 
-  @spec block_view_module!(binary) :: module | none
-  def block_view_module!(type) do
-    case lookup_block_view_module!(type) do
-      {:error, :does_not_exist, module} ->
-        raise RuntimeError,
-          message: "The application configured :block_types, but #{module} does not exist."
+  @spec block_view_module(binary) :: module
+  def block_view_module(type) do
+    type
+    |> lookup_block_view_module()
+    |> maybe_issue_warning()
+    |> Utils.module_or_fallback(UnrecognizedBlockView)
+  end
 
-      {:ok, module} ->
-        module
+  @spec lookup_block_view_module(binary) :: module_lookup_result
+  defp lookup_block_view_module(type) do
+    with {:ok, config} <- Config.fetch_config(),
```

Comment:
I think this is a perfect use case for `with`. I also learned that the `else` clause is optional and that forwarding all errors to the caller is actually a pattern that I kind of like... but it does have to be documented carefully (and calling into fallible 3rd party libraries would complicate things a lot).